tests/cases/fourslash/goToDefinitionJsModuleNameAtImportName.ts

```diff
@@ -3,10 +3,11 @@
 // @allowJs: true
 
 // @Filename: /foo.js
-//// /*moduleDef*/class Blah {
+//// /*moduleDef*/function notExported() { }

```

Comment:
this gotoDef test 

appears to show that goto-def on the imported module symbol jumps to the single export of the module,
but actually shows that goto-def on the imported module symbol jumps to the very beginning of the module.

I added a non-exported function to make this clearer.